core-plugins/widgets/XMLReader-batchsource.json

```diff
@@ -0,0 +1,78 @@
+{
+  "metadata": {
+    "spec-version": "1.0"
+  },
+  "configuration-groups": [
+    {
+      "label": "XML Reader Batch Source Properties",
+      "properties": [
+        {
+          "widget-type": "textbox",
+          "label": "Reference Name",
+          "name": "referenceName"
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Path",
+          "name": "path"
+        },
+        {
+          "widget-type": "textbox",
+          "label": "File Pattern",
+          "name": "pattern"
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Node Path",
+          "name": "nodePath"
+        },
+        {
+          "widget-type": "select",
+          "label": "Action After Processing File",
+          "name": "actionAfterProcess",
+          "widget-attributes": {
+            "values": [
+              "",
```

Comment:
Instead of empty string, we could have something explicit as "NONE" or something similar. Might not look good when viewed in UI.